src/ape/api/networks.py

```diff
@@ -357,6 +357,18 @@ def get_network_data(self, network_name: str) -> Dict:
 
         return data
 
+    def get_proxy_info(self, address: AddressType) -> Any:
+        """
+        Information about a proxy contract such as proxy type and implementation address.
+
+        Args:
+            address (str): The address of the contract.
+
+        Returns:
+            Any: ProxyInfo or None if the contract does not use any known proxy pattern.
```

Comment:
Can return type instead be `Optional[ProxyInfo]`?

Reply:
It can, but ProxyInfo can vary by ecosystem

Reply:
Added ProxyInfoAPI